### PR TITLE
Feat/session

### PR DESCRIPTION
**Pull request by DavitBakhutashvili**

### CODE REVIEW

File: apps/ui/src/components/ComponentsWrapper/SessionComponentWrapper.tsx
Comment (levanion) [inline apps/ui/src/components/ComponentsWrapper/SessionComponentWrapper.tsx:1]: What is the difference between SessionComponentWrapper and ComponentsWrapper?

File: apps/ui/src/pages/sessions/Session.tsx [hunk 1]
```diff
@@ -0,0 +1,340 @@
...
+
+import SessionDropdown from './SessionDropdown'
+import { useSchedules } from 'pages/Schedule/useSchedules'
+import { useAgentsService } from 'services/agent/useAgentsService'
+const Sessions = () => {
+  const { t } = useTranslation()
+  const { data: chatsData } = useChatsService()
+
+  const { schedules } = useSchedules()
+  const { data: agentsData } = useAgentsService()
```
Comment (levanion) [inline apps/ui/src/pages/sessions/Session.tsx:32]: Please create useSession.ts file and put this part there.

File: apps/ui/src/pages/sessions/Session.tsx [hunk 2]
```diff
@@ -0,0 +1,340 @@
...
+
+  const agentOptions = agentsData?.map((agent: any) => {
+    return { value: agent.agent.name, label: agent.agent.name }
+  })
+
+  const filteredData = mappedData?.filter(
+    (row: { name: string; agent_name: string; team_name: string }) =>
+      row.name.toLowerCase().includes(searchText.toLowerCase()) ||
+      row.agent_name.toLowerCase().includes(searchText.toLowerCase()),
+  )
```
Comment (levanion) [inline apps/ui/src/pages/sessions/Session.tsx:56]: same here

File: apps/ui/src/Route.tsx [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { Route as Router, Routes } from 'react-router-dom'
 import About from './pages/About'
 
-import Sessions from './pages/Sessions'
+import Sessions from './pages/sessions/Session'
```
Comment (okradze) [inline apps/ui/src/Route.tsx:4]: why sessions page is lowercase?

File: apps/ui/src/pages/sessions/Session.tsx [hunk 4]
```diff
@@ -0,0 +1,340 @@
...
+  const [searchText, setSearchText] = useState('')
+
+  const mappedData = chatsData?.map((chat: any) => ({
+    id: chat?.id,
+    name: chat?.name,
+    agent_name: chat?.agent?.agent?.name,
+    team_name: chat?.team?.team?.name,
+  }))
+
+  const scheduleOptions = schedules?.map((schedule: any) => ({
```
Comment (okradze) [inline apps/ui/src/pages/sessions/Session.tsx:43]: use `ScheduleWithConfigs` type from `types`

File: apps/ui/src/pages/sessions/Session.tsx [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+    value: schedule.schedule.name,
+    label: schedule.schedule.name,
+  }))
+
+  const agentOptions = agentsData?.map((agent: any) => {
```
Comment (okradze) [inline apps/ui/src/pages/sessions/Session.tsx:48]: use `AgentWithConfigs` from `types`

File: apps/ui/src/pages/sessions/Session.tsx [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+    background: ${({ theme }) => theme.body.textColorPrimary};
+  }
+
+  .css-ugu73m-placeholder {
+    color: ${({ theme }) => theme.body.textColorSecondary};
+  }
+
+  .css-1mm2o3n-indicatorContainer {
+    color: ${({ theme }) => theme.body.textColorSecondary};
+  }
```
Comment (okradze) [inline apps/ui/src/pages/sessions/Session.tsx:289]: what are those css class names?

Comment (okradze) [reply apps/ui/src/pages/sessions/Session.tsx:289]: Seems generated css class names

File: apps/ui/src/pages/sessions/SessionDropdown.tsx [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+  .dropdown-wrapper.primary__wrapper.css-7xl64p-container {
+    border: ${({ theme }) => theme.body.sessionDropdownBorder} !important;
+    height: 44px;
+    border-radius: 8px;
+  }
+  css-99wu5k {
+    path {
+      fill: ${({ theme }) => theme.body.iconColor};
+    }
+  }
```
Comment (okradze) [inline apps/ui/src/pages/sessions/SessionDropdown.tsx:146]: same here

**Review by okradze — CHANGES_REQUESTED** [review]

File: apps/ui/src/pages/sessions/columnConfig.tsx [hunk 8]
```diff
@@ -0,0 +1,199 @@
...
+  StyledDeleteIcon,
+  StyledEditIcon,
+  StyledEyeOpenIcon,
+} from 'pages/TeamOfAgents/TeamOfAgentsCard/TeamOfAgentsCard'
+import { useChatsService } from 'services/chat/useChatsService'
+import { useDeleteChatService } from 'services/chat/useDeleteChatService'
+import { ToastContext } from 'contexts'
+
+type CellProps = {
+  value: any
```
Comment (okradze) [inline apps/ui/src/pages/sessions/columnConfig.tsx:21]: fix any
it's Nullable<string>

File: apps/ui/src/components/ComponentsWrapper/SessionComponentWrapper.tsx
Comment (DavitBakhutashvili) [reply apps/ui/src/components/ComponentsWrapper/SessionComponentWrapper.tsx:1]: At this point, we don't need SessionComponentWrapper so I removed it.

File: apps/ui/src/pages/sessions/Session.tsx (hunk 1 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/pages/sessions/Session.tsx:32]: I created it and transferred it.

File: apps/ui/src/pages/sessions/Session.tsx (hunk 2 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/pages/sessions/Session.tsx:56]: I created it and transferred it.

File: apps/ui/src/Route.tsx (hunk 3 above)
Comment (DavitBakhutashvili) [reply apps/ui/src/Route.tsx:4]: It was a mistake and I corrected it.

**Pull request merged**